test/TestGrains/MethodInterceptionGrain.cs

```diff
@@ -97,6 +98,53 @@ public Task<object> Invoke(MethodInfo methodInfo, InvokeMethodRequest request, I
         public Task<string> GetInputAsString(bool input) => Task.FromResult(input.ToString(CultureInfo.InvariantCulture));
 
         public Task<int> GetBestNumber() => Task.FromResult(38);
+    }
+
+    public class GrainCallFilterTestGrain : Grain, IGrainCallFilterTestGrain, IGrainCallFilter, IGrainInvokeInterceptor
+    {
+        private const string Key = GrainCallFilterTestConstants.Key;
+        private IGrainCallContext context;
+
+        public async Task<string> Execute(bool early, bool mid, bool late)
+        {
+            if (late)
+            {
+                this.context.Arguments[2] = false;
+                await this.context.Invoke();
```

Comment:
I'll refactor the tests for the other grains which use the deprecated method and put a note here.